### ISSUE TITLE
Bug: Pipeline Status Stuck on "Deleting" After Deletion the pipeline

### ISSUE THREAD

**Issue by Neha130** [devtron-labs/devtron#5775]
### 📜 Description

encountering an issue where after deleting a pipeline, the UI continues to show the pipeline status as "Deleting"

### Affected areas

Other CRITICAL functionality

### Additional affected areas

Other CRITICAL functionality

### Prod/Non-prod environments?

Prod

### Is User unblocked?

None

### How was the user un-blocked?

None

### Impact on Enterprise

 the pipeline status remains as "Deleting" even after deleting the pipeline

### 👟 Steps to replicate the Issue

NA

### 👍 Expected behavior

 The pipeline should be removed from the UI, or if deletion is still in progress, the status should accurately reflect that.

### 👎 Actual Behavior

The pipeline status remains as "Deleting" even after deleting the pipeline

### ☸ Kubernetes version

1.30

### Cloud provider

<details>
GCP
</details>

### 🌍 Browser

Chrome

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find any similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/devtron-labs/devtron/blob/main/CODE_OF_CONDUCT.md)

**Comment by Ash-exp** [comment]
Duplicate of https://github.com/devtron-labs/devtron/issues/5774
